characters/turkeyleg.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import pygame
+
+from utils.sprite import Sprite, Collider
+from utils.collision import CollisionEffect
+
+
+IMAGE_PATH = os.path.join('images', 'turkeyleg.png')
+IMAGE = pygame.image.load(IMAGE_PATH).convert_alpha()
+
+
+class TurkeyLeg(Sprite):
+    def __init__(self, x, y, angle, rotational_velocity):
+        super().__init__(
+            image=IMAGE,
+            x=x,
+            y=y,
+            angle=angle,
+            velocity=0,
+            move_x=0,
+            move_y=0
+        )
+        self.rotational_velocity = rotational_velocity
+        self.destroyed = False
+
+    def set_collider(self):
+        self.collider = Collider(self, effect=CollisionEffect.Halt, reaction=self.collided)
```

Comment:
at least set `self.collider = None` in `__init__` so the property always exists

Reply:
set_collider is called from `__init__` in `Sprite` and all subclasses of sprite call `super()` in `__init__`

Reply:
really i should just change this to return the value and hardcode to set to `self.collider` in `Sprite`